liquibase-core/src/main/java/liquibase/database/core/SybaseDatabase.java

```diff
@@ -276,14 +294,14 @@ public int getDatabaseMinorVersion() throws DatabaseException {
         try {
             return getConnection().getDatabaseMinorVersion();
         } catch (UnsupportedOperationException e) {
-        	Scope.getCurrentScope().getLog(getClass())
-    			.warning("Your JDBC driver does not support getDatabaseMajorVersion(). Consider upgrading it.");
+            Scope.getCurrentScope().getLog(getClass())
+                    .warning("Your JDBC driver does not support getDatabaseMajorVersion(). Consider upgrading it.");
```

Comment:
The warning message here is incorrect; it should be `getDatabaseMinorVersion()`, not `getDatabaseMajorVersion()`.